Subtitle 1.srt

```diff
@@ -0,0 +1,31 @@
+1
+01:00:18,625 --> 01:00:20,041
+<b>My name is Lucy Nyirenda</b>
+
+2
+01:00:21,208 --> 01:00:23,291
+<b>I stay at Mapalwe Village, Senior group Msiwa</b>
+
+
+3
+01:00:25,291 --> 01:00:27,083
+<b>I thank K2 TASO AND B.T.M</b>
+
+4
+01:00:27,958 --> 01:00:31,333
+<b>For donating  Chlorine that help fight diarhiea disease</b>
+
+5
+01:00:31,500 --> 01:00:34,500
+<b>Subtitle</b>
+
+6
+01:00:34,500 --> 01:00:39,500
+<b>In our village we dont have boreholes</b>
+
+7
+01:00:39,625 --> 01:00:45,833
+<b>I am begging that , if there can be a chance of borehole </b>
+<b>that should be drilled here because currently we are </b>
+<b>drinking together with animals . Thank you</b>

```

Comment:
```suggestion
<b>I am pleading for support so that a borehole </b>
<b>can be drilled here, because currently we are</b>
<b>drinking together with animals. Thank you.</b>

```